src/main/java/org/folio/innreach/domain/service/impl/FolioTenantService.java

```diff
@@ -0,0 +1,22 @@
+package org.folio.innreach.domain.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.stereotype.Service;
+
+@Log4j2
+@Service
+@RequiredArgsConstructor
+public class FolioTenantService {
+
+  private final SystemUserService systemUserService;
+
+  public void initializeTenant() {
+    try {
+      systemUserService.prepareSystemUser();
+    } catch (Exception e) {
```

Comment:
Not sure the exception should be logged only and not propagated up to the caller. My understanding that tenant initialization should fail completely if this case